Assets/LeapMotionModules/ElementRenderer/Scripts/Features/Tint/LeapGuiTintFeature.cs

```diff
@@ -0,0 +1,16 @@
+﻿using UnityEngine;
+
+[AddComponentMenu("")]
+[LeapGuiTag("Tint")]
+public class LeapGuiTintFeature : LeapGuiFeature<LeapGuiTintData> {
+  public const string FEATURE_NAME = LeapGui.FEATURE_PREFIX + "TINTING";
+
+#if UNITY_EDITOR
+  public override void DrawFeatureEditor(Rect rect, bool isActive, bool isFocused) { }
+
+  public override float GetEditorHeight() {
+    return 0;
```

Comment:
The feature class controls the global settings, and is visible on the LeapGui component.  The data class controls the per-element settings, and is visible on the LeapGuiElement component.  

Whenever a gui element becomes a child of a gui, the gui manually goes through elements and makes sure that they have the data needed to support the features the gui has.